Firewall nat warning (#255)
UKCloud/documentation: articles/vmware/vmw-gs-build-vm-vcd91.md

The edge gateway is possibly the most complex part of the VDC because of its high level of functionality. The following steps show you how to quickly configure the edge gateway to enable you to access the internet from a VM.

> [!NOTE]

> > NAT rules will only work if firewall is enabled. For security reasons, the firewall should always be enabled.

Reviewer: ```suggestion
> NAT rules only work if the firewall is enabled. For security reasons, you should ensure that the firewall is always enabled.
```
